src/components/Shared/Leaflet/Markers.js

```diff
@@ -8,6 +8,8 @@ import iconShadow from 'leaflet/dist/images/marker-shadow.png';
 
 const { GEO_LOCATION } = MAP;
 
+// TODO: Why is this deleted?
```

Comment:
Maybe it doesn't have to be deleted, I saw it in some sample, and doesn't make much sense, but it worked and I just left it there.

Reply:
It broke eslint and I had to specifically ignore the rule for this line. I would vote to remove all three of these lines actually. What do you think?

Reply:
I tried removing that line, and it left the map on doctor card with missing marker: 
![image](https://user-images.githubusercontent.com/319826/145035217-575bf18d-ea9d-4e1b-a205-96418093d979.png)
so i guess it is still needed :)

Reply:
...and the name starting with underscore out of our control, making the eslint rule disabling the only option.